client-apis/encrypt.md

```diff
@@ -65,7 +65,10 @@ The plaintext to encrypt.
 
 See [encryption context](../framework/structures.md#encryption-context).
 
-The encryption context MUST NOT contain an entry with a [reserved key value](../framework/structures.md#encryption-context).
+The prefix `aws-crypto-` is reserved for internal use by the AWS Encryption SDK;
+see the [the Default CMM spec](default-cmm.md) for one such use.
```

Comment:
I think we should change the default CMM
to error if the provided EC includes `aws-crypto-public-key`.
Since we are removing the MUST from the EC section,
I would just add it there

Reply:
To clarify, we'd specify the reserved prefix in encrypt but ALSO have the default CMM check for this value explicitly, just in case the CMM is used outside of encrypt? I'm fine with that.

Reply:
Yes, I'm saying the default CMM MUST check.
Mostly to keep people safe if they try and pass this value in the EC on encrypt.
(otherwise the default CMM would clobber the value silently)

Reply:
> Mostly to keep people safe if they try and pass this value in the EC on encrypt.
> (otherwise the default CMM would clobber the value silently)

Just to clarify for the record, people won't be able to pass this value in the EC on encrypt because that is precisely where the `aws-crypto-` prefix will be rejected. But I agree the default CMM should still defend against this independently to guard against this in other potential contexts.

Reply:
Yes, better said. 

>to guard against this in other potential contexts.

I think that the primary case is CMM composition. If someone wants to compose and extend the default CMM behavior.